### ISSUE TITLE
Multimedia Player - Video: Closed captions

### ISSUE THREAD

**Issue by Wenifred** [wet-boew/wet-boew#6592]
The closed captions do not seem to be working on our videos any more. The black box pops up but the text doesn't show.  I am seeing the same thing on your multimedia player page: http://wet-boew.github.io/v4.0-ci/demos/multimedia/multimedia-en.html

**Comment by LaurentGoderre** [comment]
Oooh the damage one missign space can do.....Thanks for bringing it up!

**Comment by LaurentGoderre** [comment]
It's fixed in the latest code.

**Comment by Wenifred** [comment]
:) Perfect thanks!
